app/presenters/member_presenter.rb

```diff
@@ -18,4 +18,18 @@ def attending?(event)
   def subscribed_to_newsletter?
     opt_in_newsletter_at.present?
   end
+
+  def pairing_details_array(role, note)
+    role.eql?('Coach') ? coach_pairing_details : student_pairing_details(note)
+  end
+
+  private
+
+  def coach_pairing_details
+    [newbie?, full_name, 'Coach', 'N/A', skill_list.to_s]
```

Comment:
guessing `full_name` and `skill_list` work due to being defined on `member`?

Should `skill_list.to_s` be `skill_list.join`?

Reply:
Yes, they are delegated to Member.

`skill_list` is a taggable object so `to_s` will  return a comma separated list of all skills.